include/galsim/SBInterpolatedImage.h

```diff
@@ -29,109 +29,29 @@
 
 namespace galsim {
 
-    /**
-     * @brief A Helper class that stores multiple images and their fourier transforms
-     *
-     * One of the ways to create an SBInterpolatedImage is to build it from a
-     * weighted sum of several component images.  The idea is that the component
-     * images would be constant, but the weights might vary across the field of view.
-     * (E.g. they could be principal components of the PSF).
-     *
-     * This class stores those images along with helpful derived information
-     * (most notably, the fourier transforms), so that each SBInterpolatedImage
-     * doesn't have to recalculate everything from scratch.
-     */
-    class MultipleImageHelper
+    class SBInterpolated : public SBProfile
```

Comment:
I don't see the point of this extra level of indirection.  The `SBInterpolated` class doesn't seem to provide any significant code reuse.  It and its Impl class both have trivial constructors.  And the only function that it defines is `getKInterp`, which is also trivial.  So I think it would be better to just have both `SBInterpolatedImage` and `SBInterpolatedKImage` derive directly from `SBProfile`.